src/docs/doc.yaml

```diff
@@ -36,19 +30,116 @@ paths:
                   type: string,
                 password1:
                   type: string,
+            examples:
+              'SLIGHTLY BAD PATH':
```

Comment:
what's the point of having two different bad entries? aren't they going to give the same result?

Reply:
They return different results. One only checks the format; the other checks the database.

Reply:
okay